pkg/daemon/ceph/osd/volume.go

```diff
@@ -127,9 +146,29 @@ func (a *OsdAgent) initializeBlockPVC(context *clusterd.Context, devices *Device
 				"--data",
 				deviceArg,
 			}...)
+
 			// execute ceph-volume with the device
+			op, err := context.Executor.ExecuteCommandWithCombinedOutput(false, "", baseCommand, immediateExecuteArgs...)
+			if err != nil {
+				cvLogFilePath := path.Join(cvLogDir, "ceph-volume.log")
```

Comment:
nit: how about a separate method to cat the log?

Reply:
How critical is the c-v debug log for troubleshooting? We probably get the details from the normal log most of the time. Instead of appending it on failure, how about if we always log it if the `ROOK_LOG_LEVEL=debug` in the operator env vars? This debug level could be passed to the prepare job and then we could just always log with debug when that flag is set. 

Or if the debug log is really useful on failure, we could write it on failure, or when debug level is set.

Reply:
The debug log is already printed on failure. I don't think it's worth activating it when `ROOK_LOG_LEVEL=debug`. Since its footprint is low we should always have it turned on.
Also, most of the time, if failure appears during prepare, it's better to cash debug log during the first run.
If we don't, imagine all the steps we need to do:

* delete the cluster
* run the op with `ROOK_LOG_LEVEL=debug`
* trash the devices
* create a new cluster

This seems a bit too much as a repro.

Ok for the function to cat.

Reply:
Sounds good to capture the debug log upon failure for the OSD on PVC. For the non-PVC case it also seems like a good thing to capture the debug log if the debug log flag is set. It could certainly be a separate PR though.